pallets/thea-council/src/tests.rs

```diff
@@ -110,6 +111,24 @@ fn test_claim_membership_with_unregistered_pending_member_returns_not_pending_me
 	})
 }
 
+#[test]
+fn get_expected_votes_test() {
+	new_test_ext().execute_with(|| {
+		// at most 10 council members allowed
+		for i in 2..11 {
+			// we start with 1 and it can go up to 10
+			let members_vec: Vec<u64> =
+				(1u64..=i).into_iter().enumerate().map(|(n, _)| n as u64 + 1).collect();
+			let members = BoundedVec::try_from(members_vec).unwrap();
+			<ActiveCouncilMembers<Test>>::put(members.clone());
+			// we check if we have more than half of actual council members always
+			let expected: u64 =
+				TheaCouncil::get_expected_votes().saturated_into::<u64>().saturating_mul(2);
+			assert!(expected > i);
```

Comment:
Could we hardcode expected values and assert that `TheaCouncil::get_expected_votes().saturated_into::<u64>().saturating_mul(2);` will be the same as for example 4 for the first iteration and so on? because having regression and successfully pass this test with only this assertion looks like have bigger chance then if assert against already predefined results.

Reply:
that might work for current, at most, 10 council members but will be bunch of manual work for increased value.
with current approach there's no way this test will pass if given expected value is not satisfying 51% (supermajority)... 
2 out of 3 = 2 * 2 = 4 (everything lower will fail), fails with 1 as 1*2 < 3
2 out of 4 = 2 * 2 = 4 (error as is not `>`) only 3 or above success
and so forth...